upgrade/Stage_2.md

```diff
@@ -17,11 +52,18 @@
 
 1. Repeat the previous step for each other master node **excluding `ncn-m001`**, one at a time.
 
-## Stage 2.2
+## Argo web UI
```

Comment:
Should this just be a bullet point for Stage 2.1 instead of a heading level 2? It kind of breaks the Stage 2.1 --> 2.2 flow.

Reply:
Yeah, I agree. The reason I did it this way was to avoid breaking any places where folks may have been referencing stage 2.1, 2.2, etc. But I could have done a docs search for that. Good call out.